www/ui_components/app-root.html

```diff
@@ -424,8 +424,50 @@
           type: Object,
           readonly: true,
           value: {
+            ar: {id: 'ar', dir: 'rtl'},
```

Comment:
Not for this PR but, looking at this now, I think there must be a library like ICU for JavaScript which knows the direction for all language codes?

Reply:
There are, but the list is not too long - at least according to [Wikipedia](https://en.wikipedia.org/wiki/Right-to-left#List_of_RTL_scripts). So I'm not sure that it warrants a library.